content/api/commands/wrap.md

```diff
@@ -37,7 +37,7 @@ Pass in an options object to change the default behavior of `cy.wrap()`.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`cy.wrap()` 'yields the object it was called with' </li></List>
+- `cy.wrap()` yields the object it was called with.
```

Comment:
The unsafe warning is meant to be "here's how to avoid a common pitfall, detached DOM errors", and I added it to commands that yield DOM elements.

It could potentially be unsafe to do something like `cy.wrap(cy.$$('button'))`, but if people are doing that, then they need to go re-read "Introduction to Cypress", not a specific warning on the `cy.wrap()` "yields" section.